tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -263,7 +209,10 @@ public static void main(String[] unused) throws Exception {
 
     int exitCode;
     try {
-      exitCode = Utils.executeShell(executor.taskCommand, executor.timeOut, executor.shellEnv);
+      FrameworkRuntime mlRuntime = FrameworkRuntime.get(executor.framework);
+      mlRuntime.buildTaskEnv(executor);
+      exitCode = mlRuntime.executeTaskCommand(executor);
```

Comment:
we can call `buildTaskEnv ` in `executeTaskCommand ` ?

Reply:
Done